### PR TITLE
test: try to prevent race conditions in annotations e2e tests

### PR DESCRIPTION
**Pull request by hoorayimhelping**

### CODE REVIEW

**Review by Palakp41 — APPROVED** [review]

**Review by genehynson — CHANGES_REQUESTED** [review]
I think this will make it worse

File: cypress/e2e/util/annotationsSetup.ts [hunk 1]
```diff
@@ -28,25 +28,30 @@ export const setupData = (
...
               cy.getByTestID('add-cell--button').click()
-              cy.getByTestID('selector-list schmucket').should('be.visible')
-              cy.getByTestID('selector-list schmucket').click()
-              cy.getByTestID(`selector-list m`).should('be.visible')
-              cy.getByTestID(`selector-list m`).click()
-              cy.getByTestID('selector-list v').should('be.visible')
-              cy.getByTestID(`selector-list v`).click()
+              cy.getByTestID('selector-list devbucket')
+                .should('have.length.of.at.least', 1)
+                .click()
```
Comment (genehynson) [inline cypress/e2e/util/annotationsSetup.ts:39]: chaining like this will certainly cause more race conditions

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2159)

**Review by genehynson — APPROVED** [review]
Looks great, thanks!!

**Pull request merged**